samples/modules/thrift/hello/client/hello_client.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# Copyright (c) 2023, Meta
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""Thrift Hello Client Sample
+
+Connect to a hello service and demonstrate the
+ping(), echo(), and counter() Thrift RPC methods.
+
+Usage:
+    ./hello_client.py [ip]
+"""
+
+import argparse
+import sys
+sys.path.append('gen-py')
+
+
+from hello import Hello
+from thrift.transport import TSocket
+from thrift.transport import TTransport
+from thrift.protocol import TBinaryProtocol
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
```

Comment:
There should be a compliance failure on this line because `allow_abbrev` must be set to false. @stephanosio are we sure CI is running properly for this PR?

Reply:
It has failed compliance but just isn't showing in the review tab, I have no idea why (classic github...)

Reply:
Hmm.. looks like indentation is wrong. Kind of disappointed in formatting tools recently. This was run through autopep8. Our .clang-format fails miserably at formatting C++ as well.